client/ayon_maya/plugins/publish/extract_maya_usd.py

```diff
@@ -343,6 +347,8 @@ def parse_attr_str(attr_str):
                 )
                 del options[key]
 
+        options = self.override_options(instance)
```

Comment:
Is it supposed that the argument should be dictionary?
It errors out the extractor as shown below:
```
DEBUG: Export options: {'defaultUSDFormat': 'usdc', 'stripNamespaces': False, 'mergeTransformAndShape': True, 'exportDisplayColor': False, 'exportColorSets': True, 'exportInstances': True, 'exportUVs': True, 'exportVisibility': True, 'exportComponentTags': False, 'exportRefsAsInstanceable': False, 'eulerFilter': True, 'renderableOnly': False, 'shadingMode': 'none', 'convertMaterialsTo': 'none', 'jobContext': None, 'filterTypes': None, 'staticSingleSample': True, 'worldspace': False}
DEBUG: Performing extraction ...
DEBUG: Collected objects: ['|asset_rigMain_01_:_GRP', 'asset_rigMain_01_:rigMain_out_SET', 'asset_rigMain_01_:rigMain_controls_SET', 'asset_rigMain_01_:rigMain_skeletonMesh_SET', 'asset_rigMain_01_:rigMain_skeletonAnim_SET']
DEBUG: options: <class 'list'>
Traceback (most recent call last):
  File "C:\Users\Kayla\AppData\Local\Ynput\AYON\dependency_packages\ayon_2402141620_windows.zip\dependencies\pyblish\plugin.py", line 527, in __explicit_process
    runner(*args)
  File "D:\ayon-addon_template\ayon-maya\client\ayon_maya\plugins\publish\extract_maya_usd.py", line 350, in process
    options = self.override_options(instance)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\ayon-addon_template\ayon-maya\client\ayon_maya\plugins\publish\extract_maya_usd.py", line 429, in override_options
    options["exportUVs"] = False
    ~~~~~~~^^^^^^^^^^^^^
TypeError: list indices must be integers or slices, not str
```
[publish-report-241023-19-04.json](https://github.com/user-attachments/files/17489825/publish-report-241023-19-04.json)


As it is overriding the option, shouldn't it be: 
```suggestion
        options = self.override_options(options)
```